### DISCUSSION TITLE
Add Spotify Playlist

### DISCUSSION THREAD

**Discussion by bennysweb** [Thyraz/Sonos-Kids-Controller#19] [Q&A]
Hi,
is ist possible to add some Spotify playlists? What is the search query for that? I have tried multiple things without success.

Greetings
Benny

**Comment by murmeltier08** [comment]
Nice idea! I

**Comment by Huppi1984** [comment]
Also very interested in that? Not possible right now i suppose?

**Reply by Huppi1984** [reply]
Oh i found a way to add a playlist - have to select playlist in the top left instead of audiobook and then use the URI of the playlist.

**Reply by gar00u** [reply]
Where do i have to put the Link to the playlist in  the editor to make this work?

I tried to add a Spotify Playlist, and it looks ok in the Editor, but it doesn't show up in  the UI.

Where should I see it in the UI after I added it?

**Reply by Feuer-sturm** [reply]
@gar00u In the json file a spotify playlist looks like this:

```
    {
        "type": "spotify",
        "category": "playlist",
        "id": "xxxxxxxxxxxxxxxxxxxxxxxx"
    },

```
the ID is the string after ..../playslist/.... `https://open.spotify.com/playlist/xxxxxxxxxxxxxxxxxxxxxxxx`

Currently the musicbox is off, I will try to send you a screenshot as soon as soon as possible of the UI
